keyboards/keebfront/vanguard65/info.json

```diff
@@ -0,0 +1,130 @@
+                { "label": "K19", "matrix": [1, 9], "w": 1, "x": 8.5, "y": 1 },
+                { "label": "K1A", "matrix": [1, 10], "w": 1, "x": 9.5, "y": 1 },
+                { "label": "K1B", "matrix": [1, 11], "w": 1, "x": 10.5, "y": 1 },
+                { "label": "K1C", "matrix": [1, 12], "w": 1, "x": 11.5, "y": 1 },
+                { "label": "K1D", "matrix": [1, 13], "w": 1, "x": 12.5, "y": 1 },
+                { "label": "K1E", "matrix": [1, 14], "w": 1.5, "x": 13.5, "y": 1 },
+                { "label": "K20", "matrix": [2, 0], "w": 1, "x": 15, "y": 1 },
+                { "label": "K21", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
+                { "label": "K22", "matrix": [2, 2], "w": 1, "x": 1.75, "y": 2 },
+                { "label": "K23", "matrix": [2, 3], "w": 1, "x": 2.75, "y": 2 },
+                { "label": "K24", "matrix": [2, 4], "w": 1, "x": 3.75, "y": 2 },
+                { "label": "K25", "matrix": [2, 5], "w": 1, "x": 4.75, "y": 2 },
+                { "label": "K26", "matrix": [2, 6], "w": 1, "x": 5.75, "y": 2 },
+                { "label": "K27", "matrix": [2, 7], "w": 1, "x": 6.75, "y": 2 },
+                { "label": "K28", "matrix": [2, 8], "w": 1, "x": 7.75, "y": 2 },
+                { "label": "K29", "matrix": [2, 9], "w": 1, "x": 8.75, "y": 2 },
+                { "label": "K2A", "matrix": [2, 10], "w": 1, "x": 9.75, "y": 2 },
+                { "label": "K2B", "matrix": [2, 11], "w": 1, "x": 10.75, "y": 2 },
+                { "label": "K2C", "matrix": [2, 12], "w": 1, "x": 11.75, "y": 2 },
+                { "label": "K2D", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
+                { "label": "K2E", "matrix": [2, 14], "w": 1, "x": 15, "y": 2 },
+                { "label": "K30", "matrix": [3, 0], "w": 1.25, "x": 0, "y": 3 },
+                { "label": "K31", "matrix": [3, 1], "w": 1, "x": 1.25, "y": 3 },
+                { "label": "K32", "matrix": [3, 2], "w": 1, "x": 2.25, "y": 3 },
+                { "label": "K33", "matrix": [3, 3], "w": 1, "x": 3.25, "y": 3 },
+                { "label": "K34", "matrix": [3, 4], "w": 1, "x": 4.25, "y": 3 },
+                { "label": "K35", "matrix": [3, 5], "w": 1, "x": 5.25, "y": 3 },
+                { "label": "K36", "matrix": [3, 6], "w": 1, "x": 6.25, "y": 3 },
+                { "label": "K37", "matrix": [3, 7], "w": 1, "x": 7.25, "y": 3 },
+                { "label": "K38", "matrix": [3, 8], "w": 1, "x": 8.25, "y": 3 },
+                { "label": "K39", "matrix": [3, 9], "w": 1, "x": 9.25, "y": 3 },
+                { "label": "K3A", "matrix": [3, 10], "w": 1, "x": 10.25, "y": 3 },
+                { "label": "K3B", "matrix": [3, 11], "w": 1, "x": 11.25, "y": 3 },
+                { "label": "K3C", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
+                { "label": "K3D", "matrix": [3, 13], "w": 1, "x": 14, "y": 3 },
+                { "label": "K3E", "matrix": [3, 14], "w": 1, "x": 15, "y": 3 },
```

Comment:
this is an extra key, when compared to reference material available in [product page](https://www.keebfront.com/product/vanguard65)